lnpbp-0021.md

```diff
@@ -104,57 +101,62 @@ data AttachmentType ::
     description [Ascii ^ 1..20]
 
 interface RGB21
-    global name :: RGBTypes.AssetNaming
+    -- Asset specification containing ticker, name, precision etc.
+    global spec :: RGBTypes.DivisibleAssetSpec
```

Comment:
Now we use the same type as in RGB20, so we do not need a dedicated precision field